config/initializers/devise.rb

```diff
@@ -249,8 +249,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  Devise.setup do |config|
-    config.omniauth :facebook, ENV["FB_ID"], ENV["FB_SECRET"],
+  Devise.setup do |con|
+    con.omniauth :facebook, ENV["FB_ID"], ENV["FB_SECRET"],
```

Comment:
bizarre variable name change! was it automatic? if so, avoid it with rubocop.yml (as said above)